modules/network/vpc/variables.tf

```diff
@@ -227,3 +227,18 @@ variable "firewall_rules" {
   description = "List of firewall rules"
   default     = []
 }
+
+variable "firewall_log_config" {
+  type        = object({ metadata = string })
+  description = "Firewall log configuration for Toolkit firewall rules (var.enable_iap_ssh_ingress and others)"
+  default     = null
```

Comment:
What does null value represent? I assume it would be something along the lines of "platform default". 

If my assumption is correct, I would add the statement "null will result in using the platform default"

What is the platform default. We probably don't want to document as it may change. Would it be worth setting default explicitly to EXCLUDE since we know it can cause performance issues, it is a non-destructive change, and we are all about proposing sensible defaults for hpc?

Reply:
The configuration is a bit weird and, perhaps, best to do something like OS Login settings.

- `null` = Logging OFF entirely
- `INCLUDE_ALL_METADATA` = Logging ON including "all" metadata
- `EXCLUDE_ALL_METADATA` = Logging ON excluding "all" metadata

Reply:
The performance difference has been observed when I change from `INCLUDE_ALL_METADATA` to `null`. I have not tested `EXCLUDE_ALL_METADATA`.

Reply:
Offline changes discussed, pick a "ternary logic" string with these allowed values:

- `DISABLE_LOGGING`
- `INCLUDE_ALL_METADATA`
- `EXCLUDE_ALL_METADATA`